WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostPreviewFragment.java

```diff
@@ -41,6 +47,13 @@ public static EditPostPreviewFragment newInstance(@NonNull PostModel post) {
     public void setArguments(Bundle args) {
         super.setArguments(args);
         mLocalPostId = args.getInt(ARG_LOCAL_POST_ID);
+        mSite = (SiteModel) args.getSerializable(WordPress.SITE);
+    }
+
+    @Override
+    public void onSaveInstanceState(Bundle outState) {
```

Comment:
My latest commit on the PR doesn't have this method. https://github.com/wordpress-mobile/WordPress-Android/pull/9523/files

Reply:
thanks for the heads up - addressed in 7cf3d94bb1. Will merge after Travis goes green.